src/components/Footer/index.js

```diff
@@ -1,32 +1,54 @@
-import React from 'react'
+import React, { Fragment } from 'react'
 import styled from 'styled-components'
 import { Padding } from 'styled-components-spacing'
+import { Location } from '@reach/router'
 
 import { Grid, Row, Col } from '../../components/grid'
 import { SectionTitle } from '../../components/Typography'
+import GetInTouch from '../Common/GetInTouch'
 import OfficeListing from './OfficeListing'
 import FooterLinks from './FooterLinks'
 
 export const GreyFooter = styled.footer`
   background: #232323;
 `
 
+const showGetInTouch = pathname => {
+  const path = pathname.replace(/^\/|\/$/gi, '')
+  return !['contact', 'training', 'speciality/node-js', 'join-us'].includes(
+    path
+  )
+}
+
 const Footer = () => (
-  <GreyFooter>
-    <Padding top={{ smallPhone: 3, tablet: 4 }}>
-      <Grid>
-        <Row>
-          <Col width={1}>
-            <Padding bottom={3}>
-              <SectionTitle reverse>Find us</SectionTitle>
-            </Padding>
-          </Col>
-        </Row>
-        <OfficeListing />
-      </Grid>
-    </Padding>
-    <FooterLinks />
-  </GreyFooter>
+  <Fragment>
+    <Location>
+      {({ location: { pathname } }) =>
+        showGetInTouch(pathname) ? (
+          <GetInTouch
```

Comment:
Given the urgency of the ticket I wouldn't focus on making the change but any string like that should probably come from Contentful.

Reply:
You're right. I want to make a Meta content type that will deal with things like this and the top nav and footer links etc.

Reply:
👍